### ISSUE TITLE
North lines pattern becomes skewed

### ISSUE THREAD

**Issue by pkturner** [OpenOrienteering/mapper#1870]
### Steps to reproduce
1. Create a new map, with a CRS, and with symbol set ISOM 2017-2.
2. Add an area of North lines, symbol 601.2.
3. Use the Georeferencing dialog to adjust the declination.
4. Answer "Yes" to "Do you want to rotate the map accordingly?"

### Actual behaviour
The North lines become skewed from the vertical.

### Expected behaviour
While the object positions and shapes rotate, the North lines remain parallel to the sides of the map.

### Configuration
Mapper Version: master-v20201227.4 (v0.9.4 behaves as expected.)
Operating System: MacOS 11.2 Big Sur; also seen on Linux

### Note
Direct use of the "Rotate map" dialog has similar unexpected results.

**Comment by dl3sdo** [comment]
The pattern keeps its direction if the 'adjustable per object' checkmark is removed for the symbol.

**Comment by lpechacek** [comment]
As @dl3sdo pointed out, OCAD symbol set has the North lines area non-rotatable while Mapper has them rotatable. Thinking about the pros and cons, I recalled the times when there was no North lines area and OCAD had the Vertical lines tool for the map finishing. So in the OCAD world, if you use the Vertical lines tool to draw the north lines, the lines will be skewed after map rotation. If you use the area symbol, the lines will point to the map north. If you, for whatever reason, mix areas and lines ... sigh...

In the Mapper world, no matter if you use lines or the area symbol, the lines will point in the same direction. I don't say it's "better" but the result is consistent. Just my 0.02 €.

**Comment by pkturner** [comment]
I don't have a strong notion of what the behavior should be. It's reported to bring attention to this change, and ensure that it gets some thought.

**Comment by dg0yt** [comment]
IMO the current setting in the symbol sets is wrong, at least as long as Mapper implies that magnetic north is "up".

**Comment by pkturner** [comment]
> the current setting in the symbol sets is wrong

Yes, I thought changing the default for the symbol might be a good fix.

As I understand it, ISOM no longer allows this:
![IMG_9725S](https://user-images.githubusercontent.com/24704890/108601053-8f208c00-7368-11eb-910d-977ac66eca1c.JPG)

**Comment by dg0yt** [comment]
> As I understand it, ISOM no longer allows this [...]

Even ISOM2000 stated: "The sides of the map should be parallel to the magnetic north lines." (section 2.2, last paragraph).

People may edit such maps anyway, but they may have to enable the "adjustable per objects" for all symbols. (Possible exception symbols which interfer with some physical printing screens.)

---
Side note: What are the sides of the map, anyway? Coming from radio orienteering/ARDF, I'm used to applying advanced Origami magic to printed maps, in order to produce the minimum size which lets me draw bearings over the whole competition area. And for orienteering, my Origami creates even smaller artifacts ;-)
